### DISCUSSION TITLE
Apollo supported vehicle chassis!

### DISCUSSION THREAD

**Discussion by daohu527** [ApolloAuto/apollo#15210] [Ideas]
## Supported vehicles
The following are the vehicles that **Apollo already supports**, and we will continue to add more vehicles. If you would like to add a new vehicle, we can discuss here.

| vehicle name | document | other |
|--------------|----------|-------|
| ch           |          |       |
| devkit       |          |       |
| ge3          |          |       |
| gem          |          |       |
| lexus        |          |       |
| lincoln      |          |       |
| neolix_edu   |          |       |
| transit      |          |       |
| wey          |          |       |
| zhongyun     |          |       |

## How to add a new vehicle
todo

**Comment by CesarLiu** [comment]
we have adapted apollo 5.5 on VW Passat GTE: https://github.com/fortiss/apollo/tree/dev_fortiss/modules/canbus/vehicle/fortuna
